### PR TITLE
removed button and link to profile page

### PR DESCRIPTION
**Pull request by thetif**
The profile page is not longer relevant.

### This pull request changes...

- remove the "Manage account" button from the navigation dropdown
- remove the route to go to the profile page

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
~- [ ] The change has been documented~
~- [ ] Associated OpenAPI documentation has been updated~
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/routes.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ import Dashboard from './containers/Dashboard';
 import ApdApplication from './containers/ApdApplication';
 import ApdViewOnly from './containers/viewOnly/Apd';
 import EditAccount from './containers/admin/EditAccount';
-import MyAccount from './containers/admin/MyAccount';
```
Comment (radavis) [inline web/src/routes.js:7]: We probably want to delete this container component, and any associated unit test, too.

Comment (thetif) [reply web/src/routes.js:7]: I think we will end up having a profile page, but we don't have any designs for it yet, so I wanted to leave it there for when we will need it.

**Review by jeromeleecms — APPROVED** [review]
Seems fine - tried with screen reader and keyboard nav. Tried to navigate to the old pages and the routes showed a 404 error, but you can't see the content of the error message. Worth spinning up as a separate issue?

image

**Comment by thetif** [comment]
This ticket is for updating the Manage access page https://app.zenhub.com/workspaces/cms-eapd-5a3823ad3659e94d8ee569be/issues/cmsgov/eapd/2582. It really only lets the user request another state, so we don't want to expose that yet.

File: web/src/routes.js (hunk 1 above)
Comment (radavis) [reply web/src/routes.js:7]: But, isn't that what git is for?

**Review by radavis — APPROVED** [review]
👍

**Pull request merged**